### PR TITLE
Fix header styles

### PR DESCRIPTION
**Pull request by arufanov**
Fix header styles

### CODE REVIEW

**Comment by arufanov** [comment]
@OksanaSidorova 
* lets continue to work at this branch, I add files to ignore files to exclude node_modules at future.
* I have moved images to independent directories. Could you verify it. I am not sure about promos name.

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,11 @@ target/
 !**/src/main/**/target/
 !**/src/test/**/target/
 
+# Node files
```
Comment (arufanov) [inline .gitignore:7]: @OksanaSidorova 
* this lines allow to ignore this directory and files to add 
*

File: src/main/resources/templates/fragments/main_header.html [hunk 2]
```diff
@@ -1,9 +1,12 @@
-<header class="bg-dark py-5" id="header">
+<header class="py-5" id="header">
   <div class="container px-5">
     <div class="row gx-5 justify-content-center">
-      <div class="col-lg-6">
+      <div class="col-lg-8">
         <div class="text-center my-5">
-          <h1 class="display-5 fw-bolder text-white mb-2">No terra incognita!</h1>
+          <h1 class="display-5 fw-bolder text-white mb-4 title">NO TERRA INCOGNITA!</h1>
+          <h2 class="color-green-light">No primary radar?</h2>
```
Comment (arufanov) [inline src/main/resources/templates/fragments/main_header.html:7]: @OksanaSidorova 
* can we move this information into https://github.com/a5lab/axion/blob/main/src/main/resources/templates/home/index.html.  This file have dynamic: 
```

div th:switch="${radar}">
        <div th:case="null">
          <h1>No primary radar.</h1>
          <p1><a href="/wizard/add">Create</a> the primary radar to view it at home page.</p1>
        </div>
        <div th:case="*">
          <svg id="radar"></svg>
          <h1 th:text="${radar.title}" th:id="name"></h1>
          <p th:text="${radar.description}" th:id="description"></p>

```

File: src/main/resources/templates/layouts/radar.html [hunk 3]
```diff
@@ -2,15 +2,17 @@
...
+    <th:block th:insert="~{fragments/google_analytics.html}"/>
+    <th:block th:insert="~{fragments/meta_information.html}"/>
+    <th:block th:replace="${metas}"/>
   </head>
   <body>
-    <navbar th:insert="~{fragments/main_navbar.html}"></navbar>
-    <header th:insert="~{fragments/main_header.html}"></header>
-    <div th:replace="~{fragments/system_flash.html}"></div>
-    <div th:replace="~{fragments/main_tabbar.html}"></div>
+    <div id="main-header-section">
```
Comment (arufanov) [inline src/main/resources/templates/layouts/radar.html:10]: @OksanaSidorova 
* i expand you changes at home layout to others layouts. I am not sure: should or not system_flasy and main_tabbar be included into main-header-sectoin. Could you review this changes. Take a loot at tab menu at http://127.0.0.1:8080/settings/tenants
* will we have not main-menu-section?

File: .gitignore (hunk 1 above)
Comment (OksanaSidorova) [reply .gitignore:7]: @arufanov ok

File: src/main/resources/templates/fragments/main_header.html (hunk 2 above)
Comment (OksanaSidorova) [reply src/main/resources/templates/fragments/main_header.html:7]: @arufanov 
We have this information in the main-header-section, if we move it to this file, then the styles for this section will not be applied.

File: src/main/resources/templates/layouts/radar.html (hunk 3 above)
Comment (OksanaSidorova) [reply src/main/resources/templates/layouts/radar.html:10]: @arufanov 
 - system_flash and main_tabbar are not in the main-header-section. 

> will we have not main-menu-section?
 - what do you mean? Not sure that I've understood correctly

**Comment by OksanaSidorova** [comment]
@arufanov 

> lets continue to work at this branch, I add files to ignore files to exclude node_modules at future.
 - Ok
> I have moved images to independent directories. Could you verify it. 
 - Ok, I verified. 
> I am not sure about promos name.
 - I should rename?
> let align code on 2 spaces instead of tab.
 - Ok. I don't use tab. Only spaces. I think this problem with my editor. I'll try another.

**Pull request merged**